install_config/certificate_customization.adoc

```diff
@@ -343,38 +343,48 @@ For information specific to your load balancer environment, refer to link:https:
 [[configuring-custom-certificates-retrofit]]
 == Retrofit Custom Certificates into a Cluster
 
-You can retrofit custom master and custom router certificates into an existing {product-title} cluster by editing the 
-the Ansible inventory file, by default *_/etc/ansible/hosts_*, and running a playbook.
+You can retrofit custom master and custom router certificates into an existing
+{product-title} cluster.
 
 [[configuring-custom-certificates-retrofit-master]]
 === Retrofit Custom Master Certificates into a Cluster
 
-To retrofit custom certificates: 
+To retrofit custom certificates:
 
-. Edit the Ansible inventory file to set the `openshift_master_overwrite_named_certificates=true`. 
+. Edit the Ansible inventory file to set the `openshift_master_overwrite_named_certificates=true`.
 
-. Specify the path to the certificate using the `openshift_master_named_certificates` parameter. 
+. Specify the path to the certificate using the `openshift_master_named_certificates` parameter.
 +
 [source,yaml]
 ----
 openshift_master_overwrite_named_certificates=true
 openshift_master_named_certificates=[{"certfile": "/path/on/host/to/crt-file", "keyfile": "/path/on/host/to/key-file", "names": ["public-master-host.com"], "cafile": "/path/on/host/to/ca-file"}] <1>
 ----
 +
-<1> Path to a xref:configuring-custom-certificates-master[master API certificate]. 
+<1> Path to a xref:configuring-custom-certificates-master[master API certificate].
 
 . Change to the playbook directory and run the following playbook:
 +
 ----
 ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/redeploy-certificates.yml
 ----
 
+. If you use xref:configuring-custom-certificates[named certificates]:
```

Comment:
@jianlinliu, will you PTAL? (The other changes to the file are whitespace trimming.)

Reply:
Hi @kalexand-rh I think we need to add some explanation here that the above steps are actually not working for now, so have to manually update the certificate parameters in master-config.yaml. Or else it looks like all of these steps are required.

Reply:
@gpei, I thought you did have to do both. Scott said that "The playbook will ensure that the certificate files are distributed to the hosts but won't update master-config.yaml." I interpreted that to mean that you had to run the playbook to distribute the certs and then separately add the parameters to the master-config.yaml file.

Is that right?

Reply:
Ah yes, I forgot the step to distribute the certs, you're correct.